src/WebJobs.Script/Config/ScriptApplicationHostOptions.cs

```diff
@@ -31,5 +31,9 @@ public class ScriptApplicationHostOptions
         public IServiceProvider RootServiceProvider { get; set; }
 
         public bool IsStandbyConfiguration { get; internal set; }
+
+        public bool IsFileSystemReadOnly { get; set; }
+
+        public bool ScmRunFromPackageBlobExists { get; set; }
```

Comment:
I wonder if we shouldn't rename this to IsSCMRunFromPackage? Whe this is set, we've checked that the app setting is configured and the blob exists, so we know at that point that it's a valid SCM RFP scenario?

Reply:
That's correct and a better name for it, thanks! Updating.